browser/brave_ads/search_result_ad/search_result_ad_tab_helper.cc

```diff
@@ -19,7 +22,24 @@
 namespace brave_ads {
 
 namespace {
+
 AdsService* g_ads_service_for_testing = nullptr;
+
+constexpr int kBraveAdsIsolatedWorldId =
+    content::ISOLATED_WORLD_ID_CONTENT_END + 4;
```

Comment:
would it make sense to just use the cosmetic filters world ID here (`...+3`)? https://bravesoftware.slack.com/archives/C7VLGSR55/p1673046335116019

cc @pes10k

Reply:
Yes, this makes sense in order not to increase the number of World IDs. I will change it to `ISOLATED_WORLD_ID_CONTENT_END + 3;`

Reply:
done

Reply:
changed to `ISOLATED_WORLD_ID_BRAVE_INTERNAL`